README.md

```diff
@@ -116,7 +116,7 @@ AWS Elastic Beanstalk Setup (Docker)
 Configuration Settings
 ----------------------
 
-There are a few configuration settings you might want to adjust when setting up your instance of Mattermost. You can edit them in ./config/config.json or ./config/config/config_docker.json if you're running a docker instance.
+There are a few configuration settings you might want to adjust when setting up your instance of Mattermost. You can edit them in [./config/config.json](./config/config.json) or ./config/config/config_docker.json if you're running a docker instance.
```

Comment:
You might also want to add a link to the docker config as well. Also, you don't need the ./ at the start of relative links, but that's just a matter of style.